src/entries/popup/pages/messages/SignMessage/index.tsx

```diff
@@ -145,18 +147,31 @@ export function SignMessage({
     }
   }, [featureFlags.full_watching_wallets, isWatchingWallet, rejectRequest]);
 
+  if (!selectedWallet || !dappMetadata) return <Navigate to="/" />;
```

Comment:
not sure how i feel about this, these values can be undefined? 

if so we would be sending the user to home instead of rejecting the message?

Reply:
removed it, something super weird must happen to get into a state where the user is in this screen with no active dapp session, so I'm assuming this wont happen :)